try:    
    result = 45/1
except:
    print('error happend')
finally:
    print('finally hear')
print('done')